### PR TITLE
Clean up code base.

### PR DESCRIPTION
**Pull request by hotchemi**

### CODE REVIEW

**Comment by hotchemi** [comment]
@komamitsu Any thoughts?

File: src/main/java/com/treasuredata/android/TDClient.java [hunk 1]
```diff
@@ -85,16 +85,15 @@ private void setApiKey(String apiKey) {
     }
 
     private String createProjectIdFromApiKey(String apiKey) throws NoSuchAlgorithmException {
-        StringBuffer hexString = new StringBuffer();
```
Comment (komamitsu) [inline src/main/java/com/treasuredata/android/TDClient.java]: Good catch :)

**Comment by komamitsu** [comment]
@hotchemi Looks good. Thanks!

**Comment by hotchemi** [comment]
Thx!

**Pull request merged**